packages/graph/hash_graph/lib/graph/src/ontology/mod.rs

```diff
@@ -59,8 +59,8 @@ impl PersistedOntologyIdentifier {
     }
 
     #[must_use]
-    pub const fn owned_by_id(&self) -> AccountId {
-        self.owned_by_id
+    pub const fn owned_by_id(&self) -> &AccountId {
```

Comment:
It was for consistency with what I saw elsewhere, yes. Will address, I'm assuming you want `VersionedUri` returns to be a copy instead of a reference as well?

Reply:
VersionedUri is a string, so it's not `Copy` and we should continue to return a reference :) 

(Strings are quite expensive, they're an unknown size. `AccountId` is a `Uuid` which can be expressed as a `u128` so it's **very** cheap and can often have space allocated on the stack for it. Even if it doesn't have such an allocation, it's cheap to pass around.)

Reply:
Addressed return references to `AccountId` in https://github.com/hashintel/hash/pull/1197/commits/db703ba9f6b854f9396bcfd9facac34d48e3c5c0

Reply:
And thanks for explaining :)